### PR TITLE
Bug/fixing captcha

### PR DESCRIPTION
**Pull request by Soujanya2107**
Implementation of recaptcha by google.

### CODE REVIEW

File: pages/login.js [hunk 1]
```diff
@@ -11,6 +11,15 @@ import NavLinks from "../components/navbar";
 import SideBar from "../components/navbar-mobile";
 import ApiService from "../services/api";
 import "./login.scss";
+var Recaptcha=require('react-recaptcha');
```
Comment (radek1st) [inline pages/login.js:14]: most likely we need it on `sign up` page too

Comment (Soujanya2107) [reply pages/login.js:14]: Added captcha to Signup as well

**Comment by radek1st** [comment]
Thanks @Soujanya2107 good job. I'll block this one for the moment, until we are ready to go and have the backend implemented too.

**Comment by radek1st** [comment]
pausing this PR for now

**Pull request closed without merging**